ipydeps/__init__.py

```diff
@@ -290,23 +290,40 @@ def _find_overrides(packages, dep_link):
 
     return overrides
 
-def _run_get_stderr(cmd):
+def _run_get_stderr(cmd, progress=False):
     returncode = 0
     err = None
-
-    try:
-        subprocess.check_output(cmd, stderr=subprocess.PIPE)
-    except subprocess.CalledProcessError as e:
-        returncode = e.returncode
-
-        if sys.version_info.major == 3:
-            err = _bytes_to_str(e.stderr)
-        elif sys.version_info.major == 2:
-            err = e.output
+    
+    packages = cmd[5:]
```

Comment:
How do we know the package names start a position 5?

Reply:
This one I may need some help testing, since I'm not sure what adjustments to the `_pip_exec` args would result in package names being somewhere else, but this is what I was looking at with some print-debugging `_pip_exec` and `_run_get_stderr`:

<img width="977" alt="screen shot 2019-02-03 at 1 05 19 pm" src="https://user-images.githubusercontent.com/7707189/52180475-8fb34d80-27b4-11e9-9398-0243fe29d506.png">

Do you guys have any suggested ways of testing other optional arguments? Or would that depend on what `_config_options` are passed in?

Reply:
Thinking about this a bit more, there's already a nice list of packages in pip() on line 441.  It might make more sense to move the loop and progress bar code out to that level (and wrap it in a function to keep things neater).